aroquev/src/main/webapp/index.html

```diff
@@ -195,20 +207,22 @@ <h3>Previous comments</h3>
       <div id="js-comments-container"></div>
       <br>
 
-      <button onclick="deleteAllComments()">Delete all comments</button>
+      <button onclick="deleteAllComments()" class="btn delete-all-btn">Delete all comments</button>
     </div>
     
-
+    <hr>
     <div id="external-links">
-      <p><a id="github-link" href="https://github.com/aroquev00" target="_blank">Click here to see my GitHub Profile!
-          <img src="images/github-logo.png"></a></p>
-      <p><a id="linkedin-link"
+      <a id="github-link" href="https://github.com/aroquev00" target="_blank"><img src="images/github-logo.png"></a>
+      <a id="linkedin-link"
           href="https://mx.linkedin.com/in/armando-roque-a533a316b/en-us?trk=people-guest_people_search-card"
-          target="_blank">Click here to see my LinkedIn Profile! <img src="images/Linkedin-logo.png"></a></p>
+          target="_blank"><img src="images/Linkedin-logo.png"></a>
     </div>
-
-
+    <br>
```

Comment:
Use margin instead of `<br>` if your goal is to increase space between elements.

`<br>` is mainly used to force line breaks in text.

Reply:
Done, removed `<br>` tags and did the effect by setting margins in CSS rules.